modules/dr-restoring-cluster-state-about.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-2-restoring-cluster-state.adoc
+
+[id="dr-scenario-2-restoring-cluster-state-about_{context}"]
+= About restoring cluster state
+
+You can use an etcd backup to restore your cluster to a previous state. This can be used to recover from the following situations:
+
+* The cluster has lost the majority of control plane hosts, leading to etcd quorum loss and the cluster going offline.
+* An administrator has deleted something critical and must restore to recover.
```

Comment:
just minor tweak clarifying that the cluster is assumed unrecoverable without this action.

`must restore to recover the cluster.`